x-pack/plugins/code/public/monaco/textmodel_resolver.ts

```diff
@@ -52,7 +54,9 @@ export class TextModelResolverService implements ITextModelService {
     const repo = `${resource.authority}${resource.path}`;
     const revision = resource.query;
     const file = resource.fragment;
-    const response = await fetch(`../api/code/repo/${repo}/blob/${revision}/${file}`);
+    const response = await fetch(
+      chrome.addBasePath(`/api/code/repo/${repo}/blob/${revision}/${file}`)
```

Comment:
just to confirm. it's required to add `chrome.addBasePath` for `fetch`, and not required for `kfetch`.

Reply:
kfetch return the body of the request using body.asJson

the problem is that our request doesn't return json, so response will be null unfornately.